packages/react/src/components/ComposedModal/next/ComposedModal.js

```diff
@@ -6,10 +6,76 @@ import { ModalFooter } from '../ComposedModal';
 import classNames from 'classnames';
 
 import toggleClass from '../../../tools/toggleClass';
+import requiredIfGivenPropIsTruthy from '../../../prop-types/requiredIfGivenPropIsTruthy';
 
 import wrapFocus from '../../../internal/wrapFocus';
 import { usePrefix } from '../../../internal/usePrefix';
 
+export function ModalBody(props) {
+  const {
+    className,
+    children,
+    hasForm,
+    hasScrollingContent,
+    preventCloseOnClickOutside, // eslint-disable-line
+    ...other
+  } = props;
+  const prefix = usePrefix();
+  const contentClass = classNames({
```

Comment:
```suggestion
  const contentClass = cx({
```

Reply:
I wish we could enforce this with a lint rule or something